src/raty.js

```diff
@@ -707,6 +707,7 @@ class Raty {
     } else {
       target.textContent = score;
     }
+    console.log(target.value);
```

Comment:
🔥

Reply:
I removed this line, it was left after debugging